packages/dds/sequence/src/sharedString.ts

```diff
@@ -244,18 +252,15 @@ export class SharedString extends SharedSegmentSequence<SharedStringSegment> imp
         return this.mergeTreeTextHelper.getText(segmentWindow.currentSeq, segmentWindow.clientId, " ", start, end);
     }
 
-    /**
-     * @deprecated - use `getTextWithPlaceholders` instead.
-     */
-    public getTextRangeWithPlaceholders(start: number, end: number) {
```

Comment:
going to split this and the `waitIntervalCollectionChange` into a separate pr...

Reply:
actually nah, will just include here. Updating description to reflect that.